starport/pkg/cosmoscmd/root.go

```diff
@@ -162,6 +166,38 @@ func NewRootCmd(
 
 			customAppTemplate, customAppConfig := initAppConfig()
 
+			if cmd.Name() == "start" {
+				//check whether command is start or not
+				serverCtx := server.GetServerContextFromCmd(cmd)
```

Comment:
legit!

Reply:
What you mean?

Reply:
I mean it is pretty nice that we use this feature.